### DISCUSSION TITLE
FSUG Checklist for cloud native projects and products

### DISCUSSION THREAD

**Discussion by mlieberman85** [cncf/financial-user-group#30] [General]
This is a discussion on https://github.com/cncf/financial-user-group/blob/master/projects/fsug-checklist/fsug-checklist.md

Want to get additional feedback on the above, and have chat regarding ideas of stuff that maybe we should add in there. 

@siforster @jonmuk @trmiller

**Comment by mlieberman85** [comment]
Some prior art: https://vsaq-demo.withgoogle.com/

**Comment by siforster** [comment]
Consider adding further references such as:
* first three chapters of "Kubernetes Up and Running"
* [Kubernetes: An Enterprise Guide](https://www.amazon.com/Kubernetes-Enterprise-Effectively-containerize-applications/dp/1803230037/ref=sr_1_1?crid=1907212ODCP32&keywords=Kubernetes+%E2%80%93+An+Enterprise+Guide%3A+Effectively+containerize+applications%2C+integrate+enterprise+systems%2C+and+scale+applications+in+your+enterprise%2C+2nd+Edition&qid=1641314027&sprefix=kubernetes+an+enterprise+guide+effectively+containerize+applications%2C+integrate+enterprise+systems%2C+and+scale+applications+in+your+enterprise%2C+2nd+edition%2Caps%2C169&sr=8-1) by Marc Boorstein and Scott Surovich
* Kube Academy
* Upcoming book 'Anthos in Action'
* Repos such as https://github.com/ramitsurana/awesome-kubernetes
* Wider introductions such as Phoenix Project for agile transformation

**Reply by danpop-chainguard** [reply]
> Consider adding further references such as:
> 
> * first three chapters of ["Kubernetes Up and Running" ](https://www.amazon.com/Kubernetes-Running-Dive-Future-Infrastructure/dp/1492046531/ref=pd_lpo_1?pd_rd_i=1492046531&psc=1)
> * [Kubernetes in the Enterprise by Surovich](https://www.amazon.com/dp/1803230037/ref=cm_sw_r_tw_dp_4Y1GPS7F5VRM24ZDMMN5?_encoding=UTF8&psc=1)
> *[ Kube Academy](https://kube.academy/)
> * [Upcoming book 'Anthos in Action'](https://cloud.google.com/resources/anthos-in-action-ebook)
> * Repos such as https://github.com/ramitsurana/awesome-kubernetes
> * [Wider introductions such as Phoenix Project for agile transformation](https://www.amazon.com/dp/B078Y98RG8/ref=cm_sw_r_tw_dp_8EZ8B79EMHMZC8JKANJ2 )
